src/vs/workbench/browser/parts/media/compositepart.css

```diff
@@ -11,6 +11,10 @@
 	display: flex;
 }
 
+.monaco-workbench.mac.use-inline-toolbar > .part > .composite.title {
```

Comment:
Thanks for this note - this is what was causing me to make changes to `src/vs/base/browser/ui/sash/sash.css`  - fixing this fixes both